README.rst

```diff
@@ -79,6 +81,35 @@ printed instead.
                             Config file with options to customize how content
                             is generated.
 
+Python Library
+~~~~~~~~~~~~~~
+
+You can also generate STIX objects in a Python script.
+
+You can create single objects of a specified typwusing the generate function of
+an object generator:
+
+.. code-block:: python
+
+    object_generator = stix2generator.create_object_generator()
+    indicator = object_generator.generate("indicator")
+
+You can also use the language_processor object in a similar fashion as the
+command-line tool:
+
+.. code-block:: python
+
+    language_processor = stix2generator.create_default_language_processor()
+    indicator = language_processor.build_graph("Indicator.")
```

Comment:
This gives the impression that the processor returns a single object.  It actually returns a list (since prototyping language generally can result in the creation of arbitrary numbers of objects).  Maybe the result should be assigned to a more generally named variable, like "objects" or "graph" (seeing as how the method is named build_graph!).

Reply:
I wanted to convey that not giving a number will still only produce a single object, so I specified that the function returns a list with the quantity given. What do you think?

Reply:
It's true that only one object will be created, but a list is returned regardless (so you will have a length-one list in that case).  I don't think it's necessary to give such a simple example though.  Support for object counts is just one of many language features, and the readme is not the place to explain it.  Maybe give a slightly more complex example, which hints at the language's more general character, and makes more sense with a list-typed return value, e.g. `Malware targets Identity.` or something?  If you want to include a count, maybe `Malware targets 2 Identity.`?